src/css/components/Button.scss

```diff
@@ -38,243 +36,95 @@ $Button-padding: 0 scaleGrid(2);
-    color: $Button--blue-color-hover;
-  }
-
-  &.is-active {
-    background-color: $Button--blue-bg-hover;
-    color: $Button--blue-color-hover;
-  }
-
-  &.is-disabled,
-  &:disabled {
-    &,
     &:hover,
-    &:active {
-      background-color: $Button--blue-bg;
-      border-color: $Button--blue-border-color;
-      color: $Button--blue-color-hover;
+    &:focus {
+      background-color: $hover;
+      border-color: $hover;
+      color: $ts-white;
     }
-  }
-}
-
-
-// * Modifier: Button Orange
-
-$Button--orange-bg: $ts-orange !default;
-$Button--orange-bg-hover: scaleColor($Button--orange-bg, -1);
-$Button--orange-color: $ts-white;
-$Button--orange-color-hover: $ts-white;
-$Button--orange-border-color: scaleColor($Button--orange-bg, -2) !default;
-$Button--orange-border-color-hover: scaleColor($Button--orange-border-color, -1);
 
-.Button--orange {
-  background-color: $Button--orange-bg;
-  border-color: $Button--orange-border-color;
-  color: $Button--orange-color;
-
-  &:hover,
-  &:active,
-  &:focus {
-    background-color: $Button--orange-bg-hover;
-    border-color: $Button--orange-border-color-hover;
-    color: $Button--orange-color-hover;
-  }
-
-  &.is-active {
-    background-color: $Button--orange-bg-hover;
-    color: $Button--orange-color-hover;
-  }
-
-  &.is-disabled,
-  &:disabled {
-    &,
-    &:hover,
-    &:active {
-      background-color: $Button--orange-bg;
-      border-color: $Button--orange-border-color;
-      color: $Button--orange-color-hover;
+    &:active, &.is-active {
+      border-color: $active;
+      background-color: $active;
+      color: $ts-white;
     }
   }
 }
 
-
-// * Modifier: Button Negative
-
-$Button--negative-bg: $ts-white;
-$Button--negative-bg-hover: $cu-negative;
-$Button--negative-color: $cu-negative;
-$Button--negative-color-hover: $ts-white;
-$Button--negative-border-color: $cu-divider;
-$Button--negative-border-color-hover: $cu-negative--dark;
-
-.Button--negative {
-  background-color: $Button--negative-bg;
-  border-color: $Button--negative-border-color;
-  color: $Button--negative-color;
-
-  &:hover,
-  &:active,
-  &:focus {
-    background-color: $Button--negative-bg-hover;
-    border-color: $Button--negative-border-color-hover;
-    color: $Button--negative-color-hover;
-  }
-
-  &.is-active {
-    background-color: $Button--negative-bg-hover;
-    border-color: $Button--negative-border-color-hover;
-    color: $Button--negative-color-hover;
-  }
-
-  &.is-disabled,
-  &:disabled {
-    &,
-    &:hover,
-    &:active {
-      background-color: $Button--negative-bg;
-      border-color: $Button--negative-border-color;
-      color: $Button--negative-color;
-    }
-  }
-}
-
-
 // * Modifier: Button Small
 
 $Button--small-font-size: $tu-small-fontSize;
 $Button--small-height: scaleGrid(3);
 $Button--small-line-height: $Button--small-height - 3;
-$Button--small-padding: 0 $su-small;
+$Button--small-padding: 0 $su-xsmall;
```

Comment:
I think we still want this to be $su-small (scaleGrid(1)). I'm also wondering if we prefer to use the size name or the scale function for these. In the past we've tried to use the size name everywhere, but I always forget what the names mean. Using the function may be more clear.

Reply:
The functions feel more strict to me, so I think that approach resonates best with me. I'm fairly un-opinionated on this piece in particular though.

Reply:
Cool, let's go with function names then.